### PR TITLE
Feature/5949 rat submission flow/6149 warning if test already registered

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Screen to show an overwrite warning

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6149

## Screenshots
Simulator Screen Shot - iPhone 12 Pro - 2021-04-12 at 15 17 57
Simulator Screen Shot - iPhone 12 Pro - 2021-04-12 at 15 18 11

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/TestOverwriteNotice/TestOverwriteNoticeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	// MARK: - Private
+
+	private let viewModel: TestOverwriteNoticeViewModel
+	private let didTapPrimaryButton: () -> Void
+	private let didTapCloseButton: () -> Void
+
+	private func setupTableView() {
+		tableView.separatorStyle = .none
+		tableView.allowsSelection = false
+		tableView.backgroundColor =	.enaColor(for: .background)
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/TestOverwriteNotice/TestOverwriteNoticeViewController.swift:68]:
```suggestion
		tableView.backgroundColor = .enaColor(for: .background)
```

**Review by nickguendling — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA reviewed

**Review by freshking — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 2]
```diff
@@ -160,6 +160,27 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 
 	// MARK: Initial Screens
 
+	private func showOverrideTestNotice(
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:163]: Did I miss the call to show the VC somewhere?

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift (hunk 2 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:163]: not used here now - will get used in the next ticket

**Pull request merged**